news/2024/2024-07-06-osu-catch-world-cup-2024-concludes.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: post
+title: osu!catch World Cup 2024 Concludes
+date: 2024-07-06 12:00:00 +0000
+---
+
+After years of South Korean domination, this year, we finally crown a new osu!catch World Cup champion.
+
+![](/wiki/Tournaments/CWC/2024/img/cwc2024-banner.jpg)
+
+The United States, perservering after many podium finishes, manage for the first time in their history to win the osu!catch World Cup. They were the favourites coming into this tournament, and teared through their bracket to reach the Grand Finals with only a single map lost, and in the end, no one could stop them. It's only the fourth different country to win this tournament, after Chile, South Korea, and China, so it is a big day for our gamemode.
+
+## Summary
+
+![](/wiki/Tournaments/CWC/2024/img/cwc2024-podium.jpg)
+
+Congratulations to the United States for their historic first osu!catch World Cup victory, and congratulations to Argentina and Poland on their podium finishes.
+
+The United States have been the "challengers" for years at this point, never being the favourites to win it all against China and South Korea in previous years. They were, however, always a top 6 contender and more often even 2nd and 3rd. With a lot of their core members remaining the same over the years, it is truly a special moment for them to finally become champions, after so many closes finishes.
```

Comment:
having "years" twice that close to each other felt weird to me, this is totally subjective though
```suggestion
The United States have been the "challengers" for years at this point, but were never the favourites to win it all against China and South Korea. They were however, always a top 6 contender and more often even 2nd and 3rd. With a lot of their core members remaining the same over the years, it is truly a special moment for them to finally become champions, after so many closes finishes.
```